tests/buzzn_test_case.py

```diff
@@ -1,17 +1,24 @@
+import bcrypt
+
 from flask_testing import TestCase
 from setup_app import setup_app
-
+from util.database import db
 
 class TestConfig():
     """The config to be used in the test cases"""
     SECRET_KEY = 'testingkey'
     TESTING = True
     LIVESERVER_PORT = 0
     CLIENT_NAME = 'BuzznClient'
+    SQLALCHEMY_TRACK_MODIFICATIONS = False
+    SQLALCHEMY_DATABASE_URI = 'sqlite:///:memory:'
+    PASSWORD_SALT = bcrypt.gensalt()
 
 
 class BuzznTestCase(TestCase):
-    """Creates app configured to run buzzn tests."""
+    """Creates apps configures to run buzzn tests."""
```

Comment:
:) 
You mean: one app to run tests, right? I'd suggest "Creates app configured to run buzzn tests."